corehq/toggles.py

```diff
@@ -1660,6 +1660,15 @@ def _commtrackify(domain_name, toggle_is_enabled):
     relevant_environments={'icds', 'icds-new', 'india'},
 )
 
+
+ICDS_DISHA_API_V2 = StaticToggle(
```

Comment:
`nit`: This is a small tidbit, but all of this has nothing to do with DISHA. This was supposed to be a DISHA (which is an health ministry team) like API for NIC team, but even the similarity is not there as much anymore. So this might indicate non-existent relation to DISHA. And so, API_V2 is also not warranted.